docs/pages/server-access/guides/openssh.mdx

```diff
@@ -10,7 +10,11 @@ accept SSH certificates dynamically issued by a Teleport CA.
 
 Using Teleport and OpenSSH has the advantage of getting you up
 and running, but in the long run, we would recommend replacing `sshd` with `teleport`.
-We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://goteleport.com/blog/openssh-vs-teleport/)
+`teleport` SSH servers have support for multiple features that are incompatible with OpenSSH:
+
+- [Dynamically updating node labels from the output of configured commands](../../management/admin/labels.mdx)
+- [Session recording without SSH connection termination](recording-proxy-mode.mdx)
+- [Advanced session recording](bpf-session-recording.mdx) and [restricting outbound network connections in SSH sessions](restricted-session.mdx)
```

Comment:
Why are these two links included in the same bullet?

Reply:
No good reason I guess, they both use BPF but users don't care about that. Split into two bullets